### PR TITLE
Add sponsors to confirmation page

### LINKED ISSUE #132: Add sponsors to confirmation page
- [ ] extend the `Landing` component used in `Login` and `Confirmation` to display sponsors
Screen Shot 2020-11-26 at 9 26 33 PM

### PR DESCRIPTION
**Pull request by michelleykim**
I am so sorry if there is a specific format I need to follow when making a pull request.. I just did whatever....

## Change Summary

> Sponsor logos added to the application confirmation page, due to the tight timeline, logos are hard coded rather than fetched from backend.

## Screenshots

confirmation page

Closes #132

### CODE REVIEW

File: src/containers/Landing/Footer.js [hunk 1]
```diff
@@ -1,18 +1,51 @@
...
 import { CenterHorizontally } from '../../components/Common'
 
+const SponsorsContainer = styled.div`
+  position: absolute;
+  width: 80vw;
+  top: 37em;
+  ${CenterHorizontally}
+  display: block;
+  text-align: center;
+  justify-content: center;
```
Comment (panjenny0) [inline src/containers/Landing/Footer.js:22]:
```suggestion
  width: 100%;
  margin-top: 38em;
  display: block;
  text-align: center;
  justify-content: center;
  align-items: center;
```

File: src/containers/Landing/Footer.js [hunk 2]
```diff
@@ -1,18 +1,51 @@
...
+`
+
+const LogoContainer = styled.div`
+  position: absolute;
+  ${CenterHorizontally}
+  z-index: 100;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  top: 2.5em;
```
Comment (panjenny0) [inline src/containers/Landing/Footer.js:32]:
```suggestion
  z-index: 100;
  display: flex;
  flex-wrap: wrap;
  align-items: center;
  justify-content: center;
  margin: 24px 0;
  top: 2.5em;
```

File: src/containers/Landing/Footer.js [hunk 3]
```diff
@@ -1,18 +1,51 @@
...
+`
+
+const SponsorLogo = styled.img`
+  max-width: 230px;
+  max-height: 45px;
+  width: auto;
+  height: auto;
+  float: left;
+  margin: 32px;
```
Comment (panjenny0) [inline src/containers/Landing/Footer.js:41]:
```suggestion
  margin: 16px 32px;
```

**Review by panjenny0 — CHANGES_REQUESTED** [review]
Added some comments to make the sponsor logos more responsive
Screen Shot 2020-12-03 at 9 31 00 PM
I know the rest of the content including the social media logos are not mobile responsive yet but @acchiang will help us with this in #155 :)

**Review by panjenny0 — APPROVED** [review]
lgtm! 
why does your diff contain so many past commits 😆

**Comment by acchiang** [comment]
> lgtm!
> why does your diff contain so many past commits 😆

maybe she rebased master on top of her branch instead of merging it into her branch hahahaha

**Review by ianmah — CHANGES_REQUESTED** [review]
Could you remove `package-lock.json` (We are using yarn)
Also if you can try to fix the commit history lool not sure if merging master into this branch will fix it?

**Review by panjenny0 — APPROVED** [review]

**Review by ianmah — APPROVED** [review]

**Pull request merged**